src/schema/profile.ts

```diff
@@ -331,5 +331,14 @@ export const resolvers = traceResolvers<unknown, AuthContext>({
 
       return getUserExperience(ctx, info, entity.id);
     },
+    removeUserExperience: async (_, { id }: { id: string }, ctx) => {
+      const toRemove = await ctx.con
+        .getRepository(UserExperience)
+        .findOneOrFail({ where: { id, userId: ctx.userId } });
+
+      await ctx.con.getRepository(UserExperience).remove(toRemove);
```

Comment:
you don't have to query, you can just do delete with where query above and that should do it

Reply:
with our other deletes we usually also silently succeed, its like removing upvote which was already removed, no need to fail since user wanted to delete it, and its gone already